src/routes.json

```diff
@@ -98,6 +98,13 @@
         "title": "DMCA"
     },
     {
+        "name": "down-deep",
```

Comment:
I think I'd rather make the route more generic / reusable for future years and simply use `/camp`.